presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -225,6 +226,11 @@ void PrestoServer::run() {
   registerShuffleInterfaceFactories();
   registerCustomOperators();
 
+  // Register Velox connector factory for iceberg.
```

Comment:
Nit : Move this closer to line 238

registerPrestoToVeloxConnector(
      std::make_unique<IcebergPrestoToVeloxConnector>("iceberg"));

So that the logic appears together.

Reply:
Hey @aditi-pandit, it is odd to mix different types of connector registrations. If we group the registrations based on the connector type, we will add more empty lines to separate each connector type.
The current grouping based on the `registration type (velox or prestissimo)` is cleaner.

Reply:
We will be adding more Velox connector factory registrations after https://github.com/facebookincubator/velox/pull/8871 lands. We can revisit this then.

Reply:
That is fine.